src/main/java/bio/terra/validation/ValidationUtils.java

```diff
@@ -8,6 +8,10 @@
 
     private static final String VALID_NAME_REGEX = "[a-zA-Z0-9_]{1,63}";
 
+    private static final Pattern VALID_EMAIL_REGEX = Pattern.compile("[a-z0-9!#$%&'*+/=?^_`{|}~-]+" +
+        "(?:.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?",
```

Comment:
I'm not sure I am reading this right. Why would the dots `.` be treated as a period and not as the pattern for "accept any one character here"?